packages/core/src/lib/normalize.ts

```diff
@@ -7,17 +7,19 @@ export async function normalizeAuditOutputs(
   const gitRoot = await getGitRoot();
 
   return audits.map(audit => {
+    const { issues = [], table } = audit.details ?? {};
     if (
-      audit.details?.issues == null ||
-      audit.details.issues.every(issue => issue.source == null)
+      // early exit to avoid object cloning
```

Comment:
Now that issues are optional I will enrich this line with table info too and only early exit if both are empty